### DISCUSSION TITLE
Adding RTSP to officially unsupported cameras

### DISCUSSION THREAD

**Discussion by alexeiw123** [fuatakgun/eufy_security#427] [Ideas]
This may be totally unachievable but I stumbled across this reddit thread that discusses modifying an android apk to enable some RTSP functionality on an unsupported doorbell model. Is there an avenue for this to be built in to the integration or perhaps bropat's add-on?

https://www.reddit.com/r/EufyCam/comments/nal27f/enabling_rtsp_on_battery_doorbell/

**Comment by fuatakgun** [comment]
Let's bring this into @bropat 's attention on eufy-security-ws repository, he is the one that can do it.

**Comment by fuatakgun** [comment]
Even bropat can do it without any modifications on APK end as WS is acting as APK here.
